content/docs/tools-ui/branding-guidelines.md

```diff
@@ -31,10 +31,10 @@ The O3DE Logo Pack also also includes the above O3DE brand guidelines PDF.
 
 [See the individual files on GitHub](https://github.com/o3de/artwork)
 
-## BlueJay Design System Logo Pack
+## Blue Jay Design System Logo Pack
 
-Download the logo assets and files for the **BlueJay Design System**.
+Download the logo assets and files for the **Blue Jay Design System**.
 
-[{{< image-width "/images/tools-ui/branding-guidelines/bjds-banner.png" "700" "A preview of the BlueJay Design Logo Pack" >}}](/files/BlueJay-Design-System-Logo-Pack.zip)  
+[{{< image-width "/images/tools-ui/branding-guidelines/o3de-bjds-branding-banner.png" "700" "A preview of the Blue Jay Design Logo Pack" >}}](/files/Blue Jay-Design-System-Logo-Pack.zip)  
```

Comment:
Hmm... this isn't working perhaps as intended. The image does not work as a download link in my browser (Chrome).
```suggestion
{{< image-width "/images/tools-ui/branding-guidelines/o3de-bjds-branding-banner.png" "700" "A preview of the Blue Jay Design Logo Pack" >}}
```

Reply:
Let's remove the text and link: [Download O3DE Branding Guidelines PDF](/files/Open-3D-Branding-Guidelines-June-2021.pdf)

and replace it with: 

O3DE Branding Guidelines available soon. (Still waiting on LF to give us their files)

Reply:
I hesitate to write "available soon" in official documentation. Instead, I'll remove the O3DE logo pack and comment out this section. I've also messaged the Marketing team to get an updated package.

Reply:
Thank you. Great suggestion.